Modernize extension creation reference and add reference to new API nuget (RN 44760) (#6053)
SkylineCommunications/dataminer-docs: develop/devguide/GQI_Extensions/GQI_Extension_API.md

---
uid: GQI_Extension_API
---

# GQI extension API

> In order to create a GQI extension, the GQI extension API is needed. This API can be referenced in two ways:

Reviewer: The original phrasing was better

```suggestion
In order to create a GQI extension, you use the GQI extension API. This API can be referenced in two ways:
```
